packages/multi-select-combo-box/test/visual/common.js

```diff
@@ -4,9 +4,9 @@ import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themab
 registerStyles(
   'vaadin-multi-select-combo-box',
   css`
-    :host([focus-ring]) ::slotted(input),
-    :host([opened]) ::slotted(input) {
-      caret-color: transparent;
+    :host([focused][focus-ring]) ::slotted(input),
+    :host([focused][opened]) ::slotted(input) {
+      caret-color: transparent !important;
```

Comment:
This is needed to override styles added in #3779 to avoid flakiness in the newly added test.